js/Homework_js_3-4.js

```diff
@@ -0,0 +1,64 @@
+var oTest = {
+  /**
+   * Method that creates <h4> tag - the title of the test
+   */
+  createTitle: function () {
+    var oTestTitle = document.createElement('h4');
+    oTestTitle.innerHTML = 'Тест по программированию';
+    document.body.appendChild(oTestTitle);
+  },
+  /**
+   * Method that creates <ul> tags - the questions of the test
+   */
+  createQuestion: function () {
+    for (var i = 1; i <= 3; i++)
+    {
+      var oTestQuestion = document.createElement('ul');
+      oTestQuestion.innerHTML = i + '. Вопрос №' + i;
+      document.body.appendChild(oTestQuestion);
+      oTestQuestion.classList.add('list-group');
+    }
+  },
+  /**
+   * Method that creates <li> tags - the answers of the test with checkboxes
+   */
+  createAnswers: function () {
```

Comment:
Pass question element as a parameter to avoid DOM tree lookup and loop for questions

Reply:
done